nearcore/src/config.rs

```diff
@@ -296,11 +298,15 @@ pub struct Config {
     /// because the previous block isn't available. The witnesses wait in the pool untl the
     /// required block appears. This variable controls how many witnesses can be stored in the pool.
     pub orphan_state_witness_pool_size: usize,
-
```

Comment:
TODO: could moving the definition of orphan_state_witness_max_size here break anything (due to serialization?)

Reply:
AFAIK we only serizalize the config to JSON, so it should be order-independent, but I'm not entirely sure.
I agree that it would be nicer to have config options for the orphan pool next to each other, but it's a cosmetic change, so maybe it's not worth risking breaking stuff for it ¯\_(ツ)_/¯

Reply:
It should be just fine. It should be even more fine given this is not in prod so even if it is breaking it won't actually break anything important. 

You can even consider putting all orphan state witness pool configs into a dedicated struct. It's something that is worth it for large features like Resharding or StateSync above, here it may be a bit of a stretch but I'll let you decide. This is definitely not backwards compatible but again, not in prod, we can do whatever.

Reply:
Thanks. Moved to the next to the other definition.